dsbapi/__init__.py

```diff
@@ -36,12 +35,12 @@ def fetch_api(self):
             "LastUpdate": current_time
         }
         # Convert params into the right format
-        params_bytestring = json.dumps(params).encode("UTF-8")
+        params_bytestring = json.dumps(params, separators=(',', ':')).encode("UTF-8")
         params_compressed = base64.b64encode(gzip.compress(params_bytestring)).decode("UTF-8")
         
         # Send the request
         json_data = {"req": {"Data": params_compressed, "DataType": 1}}
-        timetable_data = requests.post(DATA_URL, json = json_data)
+        timetable_data = requests.post(DSBApi.DATA_URL, json = json_data)
```

Comment:
Wenn schon Konstante, unbedingt in den Konstruktor!

Reply:
Wenn du meinst

Reply:
So??
```python
def __init__(self, username, password):
    self.DATA_URL = "https://app.dsbcontrol.de/JsonHandler.ashx/GetData"
    self.username = username
    self.password = password
```